phone.py

```diff
@@ -13,7 +13,10 @@ def validate_phone(self):
           - It shouldn't be smaller than 15 characters.
           - Aside from the + no other special character should be permitted (i.e: #,@,%,etc).
         """
-        pass
+        if self.phone_letters:
+            if self.phone_letters[0] == "+" and len(self.phone_letters) < 15:
+                return True
+        return False
```

Comment:
This is the required code to fix the unit test. More work is required here to fix the `test_validate_phone_doesnt_allow_special_characters`